core/src/main/scala-2.10-2.12/hedgehog/core/XCompat.scala

```diff
@@ -0,0 +1,5 @@
+package hedgehog.core
+
+trait XCompat {
+  val StreamOrLazyList = scala.collection.immutable.Stream
```

Comment:
I don't love the name `StreamOrLazyList`, but I don't necessarily have a better suggestion. `StreamLike`? `Stream2`> I don't want to bikeshed it though so unless that gives you an idea let's just stick with for now.

Reply:
I also don't like it, but don't have an alternative yet. We can leave it for now and when a better name pops up go with that.